templates/service-template.yml

```diff
@@ -489,8 +484,8 @@ parameters:
 
 - name: OBSERVABILITY_CONFIG_TAG
   displayName: Observability configuration tag
-  description: Tag or branch to use inside the configuration repo
-  value: "main"
+  description: Tag or branch in a git repository, or image tag in a container registry
+  value: "latest"
```

Comment:
Why has this been changed from `main` to `latest?

Reply:
Before it was referring to the `main` branch of the resources repository, now it's referring to the `latest` tag in the quay repo. So the change is related to the name of the tag where the latest upstream changes are available.

Reply:
This means that we changed the default to use a container image, am I correct?

Reply:
Yes, that's correct.